packages/swapper/src/swappers/CowSwapper/endpoints.ts

```diff
@@ -101,20 +101,16 @@ export const cowApi: SwapperApi = {
     if (!cowswapQuoteResponse) throw new Error('CowSwap quote data is required')
 
     const { id, quote } = cowswapQuoteResponse
-    // Note: While CowSwap returns us a quote, and we have slippageBips in the appData, this isn't enough.
-    // For the slippage actually to be enforced, the final message to be signed needs to have slippage deducted.
-    // Failure to do so means orders may take forever to be filled, or never be filled at all.
-    const quoteBuyAmount = quote.buyAmount
-
-    // Remove affiliate fees off the buyAmount to get the amount after affiliate fees, but before slippage bips
-    const buyAmountAfterAffiliateFeesCryptoBaseUnit = deductAffiliateFeesFromAmount({
-      amount: quoteBuyAmount,
+    // Note: While CowSwap returns us a quote, and we have slippageBips and `partnerFee.bps` in the appData, this isn't enough.
+    // For the min out to actually to be reliable, the final message to be signed needs to have slippage and fee bps deducted.
+    // This actually matches what CoW uses as a minimum for a given quote - failure to do so means orders may take forever to be filled, or never be filled at all.
+    const { buyAmountAfterFeesCryptoBaseUnit } = getValuesFromQuoteResponse({
```

Comment:
Single source of truth for this logic, technically nothing changed in here, logic was already correct